Formula/composer.rb

```diff
@@ -17,6 +17,14 @@ class Composer < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "4c852f706d4ead18bb1cf79a972783b169fa52ce24c348a1d5dd349db5b6f404"
   end
 
+  # Keg-relocation breaks the formula when it replaces `/usr/local` with a non-default prefix
+  pour_bottle? do
+    on_macos do
+      reason "The bottle needs to be installed into `#{Homebrew::DEFAULT_PREFIX}` on Intel macOS."
```

Comment:
Didn't we have some sort of default `reason` for this? If it doesn't cover this exact case: can that be added?

Reply:
We only have `:clt_installed` as a default reason. We can add one for this, but that'll need to wait for the next release tag to be used. We should probably merge this before then and then come back to clean this up after.

Reply:
Sounds good 👍🏻

Reply:
Homebrew/brew#11742

Reply:
#81548